From 73140f48534d00ca1e8db0a8377dc62e36382fde Mon Sep 17 00:00:00 2001
From: Oli Thompson <olit@google.com>
Date: Wed, 3 Sep 2025 02:47:18 -0700
Subject: [PATCH] Add missing import in test

Bug: 442414618
Flag: EXEMPT bugfix
Test: manually tested
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:e10fa53bb78fb1f6a10a340771c75ece796e2088)
Merged-In: I7a5dd53abf816a5e79dc29ea251686b64dc6ac34
Change-Id: I7a5dd53abf816a5e79dc29ea251686b64dc6ac34
---
 .../com/android/internal/app/IntentForwarderActivityTest.java    | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/tests/coretests/src/com/android/internal/app/IntentForwarderActivityTest.java b/core/tests/coretests/src/com/android/internal/app/IntentForwarderActivityTest.java
index 9f7b10b3112f..501074665546 100644
--- a/core/tests/coretests/src/com/android/internal/app/IntentForwarderActivityTest.java
+++ b/core/tests/coretests/src/com/android/internal/app/IntentForwarderActivityTest.java
@@ -34,6 +34,7 @@ import static org.mockito.ArgumentMatchers.nullable;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.never;
 import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-- 
2.52.0.rc1.455.g30608eb744-goog

